Ginger/GingerCoreNET/DataSource/LiteDB.cs

```diff
@@ -1203,13 +1203,23 @@ public override string AddColumnName(string colName)
             return colName;
         }
 
-        public override string UpdateDSReturnValues(string Name, string sColList, string sColVals)
+        public override string UpdateDSReturnValues(string Name, string sColList, string sColVals, string fileFullPath)
         {
```

Comment:
Instead of getting the path, ask to get the number of rows form higher level

Reply:
Updated the code please review